### PR TITLE
Feature: Update parameters name in canVote and canExecute params

### PR DESCRIPTION
**Pull request by josemarinas**
## Description

Rename `address` and `addressOrEns` to `voterAddressOrEns` or  `approverAddressOrEns`on `canVote` and `canExecute` functions 

Task: [APP-1835](https://aragonassociation.atlassian.net/browse/APP-1835)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder of the package after the [UPCOMING] title and before the latest version.
- [ ] I have tested my code on the test network.

[APP-1835]: https://aragonassociation.atlassian.net/browse/APP-1835?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/client/examples/01-tokenVoting-client/06-can-vote.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
 // Create an address list client
 const client = new TokenVotingClient(contextPlugin);
 
-const voteParams: ICanVoteParams = {
+const voteParams: CanVoteParams = {
   proposalId: "0x1234567890123456789012345678901234567890_0x0",
   address: "0x1234567890123456789012345678901234567890",
```
Comment (brickpop) [inline modules/client/examples/01-tokenVoting-client/06-can-vote.ts]: Shouldn't this field be updated now?

File: modules/client/examples.md [hunk 2]
```diff
@@ -872,7 +872,7 @@ for await (const step of steps) {
 import {
```
Comment (brickpop) [inline modules/client/examples.md:870]: If the source examples changed, this file should include them all but it doesn't

File: modules/client/examples/02-addresslistVoting-client/06-can-vote.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
 // Create an address list client
 const client = new AddresslistVotingClient(contextPlugin);
 
-const voteParams: ICanVoteParams = {
+const voteParams: CanVoteParams = {
   address: "0x1234567890123456789012345678901234567890",
```
Comment (brickpop) [inline modules/client/examples/02-addresslistVoting-client/06-can-vote.ts]: Same

File: modules/client/test/integration/tokenVoting-client/methods.test.ts [hunk 4]
```diff
@@ -213,8 +213,8 @@ describe("Token Voting Client", () => {
           expect(typeof proposalId).toBe("string");
           expect(proposalId).toMatch(/^0x[A-Fa-f0-9]{40}_0x[A-Fa-f0-9]{1,64}$/);
 
-          const params: ICanVoteParams = {
-            address: TEST_WALLET_ADDRESS,
+          const params: CanVoteParams = {
+            voterAddressOrEns: TEST_WALLET_ADDRESS,
             proposalId,
             vote: VoteValues.YES,
```
Comment (brickpop) [inline modules/client/test/integration/tokenVoting-client/methods.test.ts:219]: I think this was already discussed elsewhere, but it's weird to be asking the vote here. I guess it makes sense for vote replacement-related scenarios?

File: modules/client/examples/01-tokenVoting-client/06-can-vote.ts (hunk 1 above)
Comment (josemarinas) [reply modules/client/examples/01-tokenVoting-client/06-can-vote.ts]: yes, forgot to update it, my bad

File: modules/client/examples.md (hunk 2 above)
Comment (josemarinas) [reply modules/client/examples.md:870]: I'll regenerate it

File: modules/client/test/integration/tokenVoting-client/methods.test.ts (hunk 4 above)
Comment (josemarinas) [reply modules/client/test/integration/tokenVoting-client/methods.test.ts:219]: Yep, its weird but the contracts ask for the vote

**Review by brickpop — APPROVED** [review]

**Pull request merged**